packages/react-components/react-menu/library/src/components/MenuSplitGroup/useMenuSplitGroupStyles.styles.ts

```diff
@@ -12,9 +14,14 @@ export const menuSplitGroupClassNames: SlotClassNames<MenuSplitGroupSlots> = {
  */
 const useStyles = makeStyles({
   root: {
+    [`${menuSplitGroupMultilineAttr}`]: {
+      [`& > .${menuItemClassNames.root}:nth-of-type(2)`]: {
```

Comment:
went with a combinator since that is how the current styles are used, we should improve this in the future, but the mistake was already made

Reply:
Can't you use a CSS var there with a fallback in `MenuItem`?

Reply:
not really, I'd need to target specifically the second MenuItem. If we're going to do a mechanism for this I'd rather do it in a separate PR to remove these combinators completely